rmf_demos_tasks/rmf_demos_tasks/dispatch_action.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
```

Comment:
To me the point of the scripts in this package is to act as a convenience tool for issuing a particular task. This script however aims to be very generic while at the same time has a couple of "gotchas". For example, I can submit a list of `start` locations and the robot will be guided to each of these but it will always perform the same action at each location. We do not accept a corresponding list of actions. It also make an implicit assumption on the `use_took_sink` parameter depending on the action string. We're also assuming that the action description is a string that is serialized from a json adhering to the [PerformAction](https://github.com/open-rmf/rmf_ros2/blob/main/rmf_fleet_adapter/schemas/event_description__perform_action.json) schema.

For these reasons I'm inclined to leaving it out. But if you feel it has been helpful then I guess we can have it. But I'd recommend updating the readme to describe the parameters in more detail and also change the behaviour to only accept a single start and matching action.

Reply:
Understand. I made a slight update on the readme, to clarify that only 1 type of custom action is accepted. Also, change the script to make `--use_took_sink` as an input arg, similar to `--use_sim_time`. Note that the `action description` is actually an optional description in perform action ([here](https://github.com/open-rmf/rmf_ros2/blob/91fe9b61a901f50a81e6b1b20700ee1615964a22/rmf_fleet_adapter/schemas/event_description__perform_action.json#L12-L13)).

This generic form of `dispatch_action` script is quite handy for testing out simple custom perfomactions. An example of 
 new composed cleaning action task:
 ```
 ros2 run rmf_demos_tasks dispatch_action -s lounge -a clean -ad '{ "clean_task_name": "clean_lounge" }' --use_took_sink
 ```